### DISCUSSION TITLE
How to package a app that has a Java dependency ?

### DISCUSSION THREAD

**Discussion by a4z** [Homebrew/discussions#1834] [Writing Formulae/Casks] [answered]
As the topic states, I wonder how to package an app that has a java-vm dependency? 

I would like to package an app, https://github.com/cross-language-cpp/djinni-generator .

The App can simply be downloaded from the release page. 
It uses any available Java that is available/exposed.

Can I just ignore that dependency, or is there something to care about?
Never did a brew package, any template available for that kind of java app?

**Comment by SMillerDev** [comment, marked as answer]
You can look at something like gradle, just depend on openjdk

**Reply by a4z** [reply]
Thanks!
